roles/cifmw_snr_nhc/.yamllint

```diff
@@ -0,0 +1,37 @@
+extends: default
```

Comment:
same here. Could you add it into main dir? IMHO it should be also done in other PR.